doc/_includes/tutorials/common/creating-a-sample.md

```diff
@@ -2,14 +2,8 @@ You can add a new sample to the project if you have the project Manager role on
 
 ![New sample button]({{ site.baseurl }}/images/tutorials/common/samples/add-sample-button.png)
 
-Clicking this button will take you to the Create New Sample page.  When creating a sample, you must define the sample name (only upper and lowercase letters, numbers, and the special characters <code>!, @, #, $, %, _, -</code>, and <code>`</code> are allowed) and optionally choose an organism for the sample:
+Clicking this button will open the Create New Sample mmodal dialog.  When creating a sample, you must define the sample name (only upper and lowercase letters, numbers, and the special characters <code>!, @, #, $, %, _, -</code>, and <code>`</code> are allowed) and optionally choose an organism for the sample:
```

Comment:
Typo: mmodal

Reply:
Fixed in df1dd88a0c5918862eca7a53fb08ee99760dfc34